doc/development/deprecations.rst

```diff
@@ -96,6 +90,18 @@ Completed deprecation cycles
 
 * The ``inner_transform`` and ``config`` keyword arguments in ``qml.execute`` have been removed.
   If more detailed control over the execution is required, use ``qml.workflow.run`` with these arguments instead.
+  
+  - Deprecated in v0.41
+  - Removed in v0.42
+
+* ``op.ops`` and ``op.coeffs`` for ``Sum`` and ``Prod`` have been removed. Instead, please use
+  :meth:`~.Operator.terms`.
+
+  - Deprecated in v0.35
+  - Removed in v0.42
+
+* Specifying ``pipeline=None`` with ``qml.compile`` has been removed. 
+  A sequence of transforms should now always be specified.
```

Comment:
Are we kinda merging two working branches together here? I saw in changelog there's also some duplicate seemingly.

Reply:
Oh never mind. I was reading some outdated part lmao